### DISCUSSION TITLE
It is possible to edit the messages at runtime.

### DISCUSSION THREAD

**Discussion by iang0ld** [intlify/bundle-tools#163] [Q&A]
We have some text which it kept in a database per user. We would like to add this to i18n on login. Can this be done.  Can we access the live i18n messages?

**Comment by 0x1af2aec8f957** [comment]
Perhaps you can fulfill your needs in the following two ways:
1. When translation data is requested, [merge](https://kazupon.github.io/vue-i18n/api/#mergelocalemessage-locale-message) with the current local translation。
2. Use [i18next-vue](https://locize.com/blog/i18next-vue), and then through [i18next-http-backend](https://github.com/i18next/i18next-http-backend) Complete your remote message configuration and load.
